test/expected/plan_expand_hypertable_optimized.out

```diff
@@ -1085,3 +1085,122 @@ SELECT * FROM cte ORDER BY value;
+(9 rows)
+
+:PREFIX SELECT * FROM hyper_w_space h1 JOIN hyper_ts h2 ON h1.device_id=h2.device_id WHERE chunks_in(h1, ARRAY[104,105]) AND chunks_in(h2, ARRAY[116,117]) ORDER BY h1.value;
+                         QUERY PLAN                          
+-------------------------------------------------------------
+ Sort
+   Sort Key: h1.value
+   ->  Hash Join
+         Hash Cond: (h2.device_id = h1.device_id)
+         ->  Append
+               ->  Seq Scan on _hyper_3_116_chunk h2
+               ->  Seq Scan on _hyper_3_117_chunk h2_1
+         ->  Hash
+               ->  Append
+                     ->  Seq Scan on _hyper_2_104_chunk h1
+                     ->  Seq Scan on _hyper_2_105_chunk h1_1
+(11 rows)
+
+:PREFIX SELECT * FROM hyper h1, hyper h2 WHERE chunks_in(h1, ARRAY[1,2]) AND chunks_in(h2, ARRAY[2,3]);
+                     QUERY PLAN                      
+-----------------------------------------------------
+ Nested Loop
+   ->  Append
+         ->  Seq Scan on _hyper_1_1_chunk h1
+         ->  Seq Scan on _hyper_1_2_chunk h1_1
+   ->  Materialize
+         ->  Append
+               ->  Seq Scan on _hyper_1_2_chunk h2
+               ->  Seq Scan on _hyper_1_3_chunk h2_1
+(8 rows)
+
+SET enable_seqscan=false;
+:PREFIX SELECT time FROM hyper WHERE time=0 AND chunks_in(hyper, ARRAY[1]);
+                                   QUERY PLAN                                    
+---------------------------------------------------------------------------------
+ Append
+   ->  Index Only Scan using _hyper_1_1_chunk_hyper_time_idx on _hyper_1_1_chunk
+         Index Cond: ("time" = 0)
+(3 rows)
+
+:PREFIX SELECT first(value, time) FROM hyper h WHERE chunks_in(h, ARRAY[1]);
+                                             QUERY PLAN                                              
+-----------------------------------------------------------------------------------------------------
+ Result
+   InitPlan 1 (returns $0)
+     ->  Limit
+           ->  Merge Append
+                 Sort Key: h."time"
+                 ->  Index Scan Backward using _hyper_1_1_chunk_hyper_time_idx on _hyper_1_1_chunk h
+                       Index Cond: ("time" IS NOT NULL)
+(7 rows)
+
+\set ON_ERROR_STOP 0
+:PREFIX SELECT * FROM hyper WHERE chunks_in(hyper, ARRAY[1,2]) AND chunks_in(hyper, ARRAY[2,3]);
+psql:include/plan_expand_hypertable_chunks_in_query.sql:22: ERROR:  only one chunks_in call is allowed per hypertable
+:PREFIX SELECT * FROM hyper WHERE chunks_in(2, ARRAY[1]);
+psql:include/plan_expand_hypertable_chunks_in_query.sql:23: ERROR:  function chunks_in(integer, integer[]) does not exist at character 48
+SELECT * FROM hyper WHERE time < 10 OR chunks_in(hyper, ARRAY[1,2]);
+psql:include/plan_expand_hypertable_chunks_in_query.sql:24: ERROR:  chunks_in function must appear in the WHERE clause, needs to be on the top level and can't be combined with ORs
+SELECT chunks_in(hyper, ARRAY[1,2]) FROM hyper;
```

Comment:
I'd like to see a test for the error case when a chunk ID in `chunk_in` does not belong to the given hypertable.

Reply:
fixed